requirements.txt

```diff
@@ -0,0 +1,8 @@
+flask
+tornado
+pyJWT
+requests
+jupyter_server
+jupyterhub
+oauthenticator
+configurable-http-proxy
```

Comment:
configurable-http-proxy is not a Python package, so shouldn't be a dependency.

These can be in pyproject.toml, or you can use [dynamic config](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata) to keep these in requirements.txt, but load them as actual dependencies so `pip install .` will work